src/librustc/mir/visit.rs

```diff
@@ -159,6 +159,15 @@ macro_rules! make_mir_visitor {
                 self.super_place(place, context, location);
             }
 
+            fn visit_neoplace(
```

Comment:
hmm I *think* I imagined `NeoPlace` being used only "transiently" rather than being stored in the MIR (and thus having to be visited) -- at least until we switch over everything, of course. 

But I guess this is ok too. I'll see where you're going with it =)